services/src/main/java/org/keycloak/forms/login/freemarker/FreeMarkerLoginFormsProvider.java

```diff
@@ -29,11 +29,8 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Properties;
-import jakarta.ws.rs.core.MultivaluedHashMap;
-import jakarta.ws.rs.core.MultivaluedMap;
-import jakarta.ws.rs.core.Response;
-import jakarta.ws.rs.core.UriBuilder;
-import jakarta.ws.rs.core.UriInfo;
+
+import jakarta.ws.rs.core.*;
```

Comment:
@lexcao Please, don't use the star imports.

Reply:
Thanks, let me fix it.